backup/pvc/Makefile

```diff
@@ -186,3 +186,18 @@ endif
 	@echo "Dependencies:"
 	dep status
 	@echo
+
+# Multi-arch (buildx) support mirroring root Makefile
+PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
+
+.PHONY: docker-buildx
+docker-buildx: check-env ## Build and push multi-arch image (set IMG=quay.io/...:tag)
+	@echo "+ $@"
+	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
```

Comment:
there is no need of doing this, take a look on how we do the whole flow in the root project makefile: https://github.com/jenkinsci/kubernetes-operator/blob/bf040e1188f2798a8e88d4bf6559c57da000cde6/Makefile#L304-L364

also in this file we already have a docker-* commands, let's resume those, ty!